Composer/packages/lib/shared/src/dialogFactory.ts

```diff
@@ -90,6 +90,12 @@ const initialDialogShape = () => ({
   [SDKKinds.SendActivity]: {
     activity: '',
   },
+  [SDKKinds.BeginSkill]: {
+    botId: '=settings.MicrosoftAppId',
+    skillHostEndpoint: '=settings.skillHostEndpoint',
+    connectionName: '=settings.connectionName',
+    allowInterruptions: 'false',
```

Comment:
This should be a boolean false and not stringified false. @zhixzhan  is there a reason you made it a string?

Reply:
@srinaath Thanks. it should be boolean.